modules/openapi-generator/src/main/resources/go-experimental/api.mustache

```diff
@@ -134,14 +134,16 @@ func (a *{{{classname}}}Service) {{{nickname}}}(ctx _context.Context{{#hasParams
 	{{#queryParams}}
 	{{#required}}
 	{{#isCollectionFormatMulti}}
-	t:={{paramName}}
-	if reflect.TypeOf(t).Kind() == reflect.Slice {
-		s := reflect.ValueOf(t)
-		for i := 0; i < s.Len(); i++ {
-			localVarQueryParams.Add("{{baseName}}", parameterToString(s.Index(i), "{{#collectionFormat}}{{collectionFormat}}{{/collectionFormat}}"))
+	{
```

Comment:
Why is this change necessary? Is it to make sure that the scope of variables defined in this block is limited?

Reply:
This is totally independent of the embedded go struct enhancement, maybe I should have raised a separate PR, but it takes time to manage multiple PRs.
Without this change, there can be compilation errors and static analysis issues. This is because the block (which is now in curly braces) is repeated multiple times depending on how many query parameters there are in the OAS. 

The issue is that this line will be generated multiple times:
```
t:={{paramName}}
```